// @azure/arm-resources-subscriptions/dist-esm/src/models/index.js
/*
 * Copyright (c) Microsoft Corporation.
 * Licensed under the MIT License.
 *
 * Code generated by Microsoft (R) AutoRest Code Generator.
 * Changes may cause incorrect behavior and will be lost if the code is regenerated.
 */
/** Known values of {@link RegionType} that the service accepts. */
export var KnownRegionType;
(function (KnownRegionType) {
    /** Physical */
    KnownRegionType["Physical"] = "Physical";
    /** Logical */
    KnownRegionType["Logical"] = "Logical";
})(KnownRegionType || (KnownRegionType = {}));
/** Known values of {@link RegionCategory} that the service accepts. */
export var KnownRegionCategory;
(function (KnownRegionCategory) {
    /** Recommended */
    KnownRegionCategory["Recommended"] = "Recommended";
    /** Extended */
    KnownRegionCategory["Extended"] = "Extended";
    /** Other */
    KnownRegionCategory["Other"] = "Other";
})(KnownRegionCategory || (KnownRegionCategory = {}));
/** Known values of {@link ResourceNameStatus} that the service accepts. */
export var KnownResourceNameStatus;
(function (KnownResourceNameStatus) {
    /** Allowed */
    KnownResourceNameStatus["Allowed"] = "Allowed";
    /** Reserved */
    KnownResourceNameStatus["Reserved"] = "Reserved";
})(KnownResourceNameStatus || (KnownResourceNameStatus = {}));
